src/utils/metrics.js

```diff
@@ -75,6 +77,19 @@ let shouldShowModalNewsletter = (() => {
   return now - lastPrompt > SHOW_INTERVAL;
 })();
 
+let shouldShowModalTakeover = (() => {
+  if (onTestNewsletterPage) {
+    return true;
+  }
+  let cameFromMCOn = loadDate(FROM_MC_KEY) || 0;
```

Comment:
You can delete these lives from shouldShowModalTakeover because we don't care if you came from Mailchimp, we still want you to see the takeover.

Reply:
Updated: https://github.com/spotlightpa/poor-richard/pull/450/commits/17bfd2ecccea2f8fc4945d53975b6a5659be3b0c